src/app/projectView.hook.js

```diff
@@ -53,12 +52,26 @@ export default (
         .then(() => {
           ProjectLoadingEmitter.emit(FINISHED_LOADING_PROJECT_EVENT);
         })
-        .catch(() => {
+        .catch(targetState => {
+          let stateService = trans.router.stateService;
```

Comment:
should use `const` here. If you use an eslint integration with your editor, it'll help point out best practices like this